docs/source/delta-deletion-vectors.md

```diff
@@ -27,7 +34,7 @@ ALTER TABLE <table_name> SET TBLPROPERTIES ('delta.enableDeletionVectors' = true
 
 Deletion vectors indicate changes to rows as soft-deletes that logically modify existing Parquet data files in the <Delta> tables. These changes are applied physically when data files are rewritten, as triggered by one of the following events:
 
-* An `UPDATE` or `MERGE` command is run on the table.
+* An `UPDATE` or `MERGE` command with deletion vectors disabled (by command flag or table property) is run on the table.
```

Comment:
`DELETE` is still missing (which was my original point^^)

Reply:
Oh right yes :)